core/src/main/java/org/elasticsearch/action/ingest/delete/DeletePipelineAction.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-package org.elasticsearch.plugin.ingest.transport.delete;
+package org.elasticsearch.action.ingest.delete;
```

Comment:
Do we really need actions to all be under separate packages? This seems similar to processors. Can we just put everything that isn't a processor under o.e.ingest (although I would personally just put everything under it)?